src/main/webapp/index.html

```diff
@@ -10,10 +10,20 @@
     <div id="content">
       <h1>My Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
+      <p>Click <a href="/hello">here</a> to see content created by server code.</p> <!--Could probably make this into a link of galleries of images/messages (MVP)
+                                                                                        Or when login is created, to their uploaded images page-->
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+      <div id="image-form">
+          <form action="/upload" method="POST" enctype="multipart/form-data"> <!--Redirect to page with images?-->
+                <p>Add an image and a message!</p>
+                <textarea name="message">Type a message.</textarea>
+                <br/>
```

Comment:
Break tags are kinda hacky and not really used much any more. I'd consider a CSS flex layout on the outer div to move this stuff around.

Reply:
Alright, cool. I was planning to do more CSS work too. I assume it would also be easier to change later on?